backend/module/chaos/serverexperimentation/rtds/rtds_cache_test.go

```diff
@@ -67,20 +79,35 @@ func createLatencyExperiment(t *testing.T, upstreamCluster string, downstreamClu
 
 func mockGenerateFaultData(t *testing.T) []*experimentation.Experiment {
 	return []*experimentation.Experiment{
-		createAbortExperiment(t, "serviceA", "serviceB", 10, 404),
-		createAbortExperiment(t, "serviceC", "", 20, 504),
-		createLatencyExperiment(t, "serviceA", "serviceD", 30, 100),
-		createLatencyExperiment(t, "serviceE", "", 40, 200),
+		// Abort - Service B -> Service A (Internal)
+		createAbortExperiment(t, "serviceA", "serviceB", 10, 404, false),
```

Comment:
to add to my previous comments: the code would be easier to read if we were passing ingress/egress enums instead of false/true as the last argument of `create*Experiment` calls

Reply:
not sure if the doc strings will be needed if we start using enums - maybe you could get rid of them then (up to u)